config/deploy/staging/nginx-configmap.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-configmap
+  namespace: rubygems-staging
```

Comment:
does this namespace need to be removed? I think kubernetes-deploy adds it for the rest of the resources.

Reply:
yeah this is unnecessary since doing the `apply will add it automatically.